### PR TITLE
k8s_tasks.yml: make some changes needed for k8s

### PR DESCRIPTION
**Pull request by unclejack**
This PR brings in more changes needed for the Kubernetes installation.

### CODE REVIEW

File: roles/contiv_network/tasks/k8s_tasks.yml [hunk 1]
```diff
@@ -5,20 +5,30 @@
...
+
 - name: copy contiv cni bin to k8s plugin dir
   copy:
     src: "/usr/bin/contiv/netplugin/contivk8s"
     dest: "{{ k8s_net_plugin_bin_dir }}/contivk8s"
     mode: u=rwx,g=rx,o=rx
     remote_src: yes
     force: yes
-  notify: restart kubernetes
+  #notify: restart kubernetes
```
Comment (rhim) [inline roles/contiv_network/tasks/k8s_tasks.yml:15]: Is this for commenting out?

Comment (unclejack) [reply roles/contiv_network/tasks/k8s_tasks.yml:15]: @rhim: We'll have to disable this for now. We won't support the installation of kubernetes with these playbooks for the time being.

**Comment by gaurav-dalvi** [comment]
Could you please explain why do we need this change ? 
title of the PR does not give any idea.

**Comment by unclejack** [comment]
This is work in progress. I'm investigating some issues.

File: roles/contiv_network/tasks/k8s_tasks.yml (hunk 1 above)
Comment (neelimamukiri) [reply roles/contiv_network/tasks/k8s_tasks.yml:15]: @unclejack , can you raise a CCN issue to track re-enabling this for CI/CD.

File: roles/contiv_network/tasks/k8s_tasks.yml [hunk 2]
```diff
@@ -5,20 +5,30 @@
...
 
+- name: ensure kubernetes dirs exist
+  file:
+    path: "{{ item }}"
+    recurse: yes
+    state: directory
+  with_items:
+    - "{{ k8s_manifests_dir }}"
+    - "{{ k8s_cert_dir }}"
+    - "{{ k8s_net_plugin_config_dir }}"
```
Comment (neelimamukiri) [inline roles/contiv_network/tasks/k8s_tasks.yml]: Netplugin dir part is required. Why do we need the k8s_manifests & certs here? Netplugin wont work in an install which doesnt have these folders. And by having these created, we increase the chance that some one misconfigures the folder paths. As we depend on the content of these folders, and we dont create them, its better to fail early if they are not there.

Comment (unclejack) [reply roles/contiv_network/tasks/k8s_tasks.yml]: I've removed this change.

**Review by neelimamukiri — APPROVED** [review]

**Pull request merged**